beacon_node/execution_layer/src/engine_api/json_structures.rs

```diff
@@ -350,12 +350,14 @@ impl From<Withdrawal> for JsonWithdrawal {
 
 impl From<JsonWithdrawal> for Withdrawal {
     fn from(jw: JsonWithdrawal) -> Self {
+        // This comparison is to avoid a scenarion where the EE gives us too large a number this
+        // panics when it attempts to case to a `u64`.
```

Comment:
```suggestion
        // This comparison is done to avoid a scenario where the EE gives us too large a number and we
        // panic when attempting to cast to a `u64`.
```